tests/test_numpy_scalars.py

```diff
@@ -0,0 +1,78 @@
+import numbers
+
+import numpy as np
+import pytest
+
+from psignifit import psignifit
+from psignifit._utils import cast_np_scalar
+from .fixtures import input_data
+from psignifit.sigmoids import ALL_SIGMOID_CLASSES
+
+@pytest.mark.parametrize('num', (1, 1., np.int64(1), np.float64(1.)))
+def test_cast_np_scalar_numbers(num):
+    cast = cast_np_scalar(num)
+    assert not isinstance(cast, np.number)
+    assert isinstance(cast, numbers.Number)
+
+def test_cast_np_scalar_ndarray_noop():
```

Comment:
please 2 spaces between functions everywhere (PEP8)

Reply:
fixed in 5f66c62036dab3b87661e8827d2715407021440a